main.asm

```diff
@@ -55,13 +55,13 @@ _start:
 	jne exit_with_help
 
 	mov rax, [rsp+16] ;Directory (first) parameter
-	mov [directory_path], rax 
+	mov [rel directory_path], rax
```

Comment:
alternatively, we could put [default rel](https://github.com/yasm/yasm/wiki/AMD64#rip-relative-addressing) but maybe explicit is better than implicit?